scripts/mysqld_safe.sh

```diff
@@ -21,9 +21,11 @@ mysqld_ld_library_path=
 flush_caches=0
 numa_interleave=0
 wsrep_on=0
+simulate=0
```

Comment:
I'm not sure I like this. What's the point of all this "simulate" stuff?

The problem is that it complicates future edits of this script: if one wants to add some action, he has to handle "simulate" case too. :(